yt/yt/client/table_client/wire_protocol.h

```diff
@@ -280,13 +280,24 @@ struct IWireProtocolReader
 
 ////////////////////////////////////////////////////////////////////////////////
 
+struct TWireProtocolReaderLimits {
```

Comment:
I think it is better to name this class `TWireProtocolOptions`. It is closer to how we usually name such classes, and it is more generic if we happen to reuse this somewhere else in the stack.